packages/block-library/src/site-logo/index.php

```diff
@@ -132,7 +132,7 @@ function _sync_custom_logo_to_site_logo( $value ) {
  * @param array $old_value Previous theme mod settings.
  * @param array $value     Updated theme mod settings.
  */
-function _delete_site_logo_on_remove_custom_logo( $old_value, $value ) {
+function _gutenberg_delete_site_logo_on_remove_custom_logo( $old_value, $value ) {
```

Comment:
This isn't ideal because it means that, when these changes are merged into Core, that Core will have a function prefixed with `_gutenberg`.